pkgs/development/python-modules/python-hwdata/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hwdata
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "python-hwdata";
+  version = "2.4.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "xsuchy";
+    repo = "python-hwdata";
+    rev = "python-hwdata-${version}-1";
+    hash = "sha256-hmvxVF9LOkezXnJdbtbEJWhU4uvUJgxQHYeWUoiniF0=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  # Default hwdata database locations are hardcoded to /usr/share/hwdata
```

Comment:
```suggestion
```

We encounter hardcoded/FHS paths all the time, it is not note-worthy and the substitution already says as much.